README.adoc

```diff
@@ -178,60 +185,66 @@ The [hotspot=15-18 file=2]`handles()` method inspects the HTTP response code to
 // =================================================================================================
 == Injecting the client with dependency injection
 
-Now, instantiate the [hotspot=17-22 file=1]`SystemClient` interface and use it in the `inventory` service. If you want to connect only with the default host name, you can easily instantiate the `SystemClient` with CDI annotations. CDI injection simplifies the process of bootstrapping the client.
+Now, instantiate the [hotspot=SystemClient file=1]`SystemClient` interface and use it in the `inventory` service. If you want to connect only with the default host name, you can easily instantiate the `SystemClient` with CDI annotations. CDI injection simplifies the process of bootstrapping the client.
 
-First, you need to define the base URL of the [hotspot=17-22 file=1]`SystemClient` instance.
+First, you need to define the base URL of the [hotspot=SystemClient file=1]`SystemClient` instance.
 Configure the default base URL with the MicroProfile Config feature. This feature is enabled for you in both `server.xml` and `pom.xml` files.
 
-[role="code_command hotspot", subs="quotes"]
+[role="code_command hotspot file=0", subs="quotes"]
 ----
 #Create the configuration file.#
 `src/main/webapp/META-INF/microprofile-config.properties`
 ----
 microprofile-config.properties
-[source, properties, linenums, role='code_column']
+[source, properties, linenums, role='code_column hide_tags=copyright']
 ----
-include::finish/src/main/webapp/META-INF/microprofile-config.properties[tags=**;!copyright]
+include::finish/src/main/webapp/META-INF/microprofile-config.properties[]
 ----
 
-The config property [hotspot=1 file=0]`io.openliberty.guides.inventory.client.SystemClient/mp-rest/url` is configured to the default [hotspot=2 file=0]`\http://localhost:9080/system` base URL.
+The config property [hotspot=baseUri file=0]`mp-rest/uri` is configured to the default [hotspot=baseUri file=0]`\http://localhost:9080/system` base URL.
```

Comment:
```suggestion
The config property [hotspot=baseUri file=0]`systemClient/mp-rest/uri` is configured to the default [hotspot=baseUri file=0]`\http://localhost:9080/system` base URL.
```

This should be "systemClient/mp-rest/uri".

Reply:
Was debating whether to use the configKey value here because it wasn't being explained prior to this statement yet. The explanation comes later in the section. Fixed.

Reply:
Maybe change it to `*/mp-rest/uri` and explain that the star can be either the fully qualified interface name or the config key if the `@RegisterRestClient` annotation includes a config key.